colabs/wandb_registry/zoo_wandb.ipynb

```diff
@@ -0,0 +1,984 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/wandb/examples/blob/master/colabs/wandb_registry/zoo_wandb.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+    "<!--- @wandbcode{zoo-wandb} -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d75eb97c",
+   "metadata": {},
+   "source": [
+    "<img src=\"http://wandb.me/logo-im-png\" width=\"400\" alt=\"Weights & Biases\" />\n",
+    "\n",
+    "<!--- @wandbcode{zoo-wandb} -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f7e5952",
+   "metadata": {},
+   "source": [
+    "# Weights & Biases Registry Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "507ab87f",
+   "metadata": {},
+   "source": [
+    "The goal of this notebook is to demonstrate how you and other members of your organization can use W&B Registry to track, share, and use dataset and model artifacts in your machine learning workflows. By the end of this notebook, you will know how to use W&B to:\n",
+    "\n",
+    "1. Create [collections](https://docs.wandb.ai/guides/registry/create_collection) within the [Dataset and Model registry](https://docs.wandb.ai/guides/registry/registry_types#core-registry).\n",
+    "2. Make dataset and model artifacts available to other members of your organization, and\n",
+    "3. Download our trained model and dataset artifacts from the registry for inference\n",
+    "\n",
+    "To achieve this, we will train a neural network to identify animal classes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c88f92cd-90ff-41cc-a656-2502799a1989",
+   "metadata": {},
+   "source": [
+    "## Install and import packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b87ee0c6-4e88-4c77-9cf5-dde109bb7ebd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#!pip install wandb torch ucimlrepo scikit-learn pandas"
```

Comment:
Should be un-commented by default IMO, since a new colab runtime won't have these (likely where most notebooks are ran) and if running locally it will just display already installed packages